### PR TITLE
add category page

### PR DESCRIPTION
**Pull request by wgugulski**
Added category page with styles

### CODE REVIEW

File: src/assets/scss/App.scss [hunk 1]
```diff
@@ -118,3 +118,179 @@ th {
...
+.search{
+  padding: 20px 60px;
+  width: 300px;
+  border: 1px solid #3F3D56;
+  border-radius: 30px;
+  -webkit-transition: all .55s ease;
+-moz-transition: all .55s ease;
+-ms-transition: all .55s ease;
+-o-transition: all .55s ease;
+
```
Comment (oskargaw) [inline src/assets/scss/App.scss:185]: you can remove space here

File: src/assets/scss/App.scss [hunk 2]
```diff
@@ -118,3 +118,179 @@ th {
...
+.search:hover{
+  padding: 20px 60px;
+  width: 310px;
+  border: 1px solid #33C296;
+  -webkit-transition: all .55s ease;
+-moz-transition: all .55s ease;
+-ms-transition: all .55s ease;
+-o-transition: all .55s ease;
+
```
Comment (oskargaw) [inline src/assets/scss/App.scss:196]: remove space here

File: src/assets/scss/App.scss [hunk 3]
```diff
@@ -118,3 +118,179 @@ th {
...
+          border-radius:42px;
+          border:2px solid #33C292;
+          display:inline-block;
+          cursor:pointer;
+          color:#fff;
+          font-family:Arial;
+          font-size:20px;
+          padding:5px 20px;
+          text-decoration:none;
+
```
Comment (oskargaw) [inline src/assets/scss/App.scss:236]: remove space here

File: src/assets/scss/App.scss [hunk 4]
```diff
@@ -118,3 +118,179 @@ th {
...
+          font-family:Arial;
+          font-size:20px;
+          padding:5px 20px;
+          text-decoration:none;
+          
+      }
+    }
+
+  @media (max-width: 768px) {
+
```
Comment (oskargaw) [inline src/assets/scss/App.scss:241]: remove space here

File: src/assets/scss/App.scss [hunk 5]
```diff
@@ -118,3 +118,179 @@ th {
...
+          padding:5px 20px;
+          text-decoration:none;
+          
+      }
+    }
+
+  @media (max-width: 768px) {
+      
+      .bigBox{
+
```
Comment (oskargaw) [inline src/assets/scss/App.scss:243]: remove space here

File: src/assets/scss/App.scss [hunk 6]
```diff
@@ -118,3 +118,179 @@ th {
...
+          grid-row: auto auto;
+          grid-column-gap: 10px;
+          grid-row-gap: 10px;
+      }
+      .text-to-change{
+          text-align: center !important;
+      }
+
+      .myButton {
+
```
Comment (oskargaw) [inline src/assets/scss/App.scss:259]: remove space here

File: src/assets/scss/App.scss [hunk 7]
```diff
@@ -118,3 +118,179 @@ th {
...
+          grid-row-gap: 10px;
+      }
+      .text-to-change{
+          text-align: center !important;
+      }
+
+      .myButton {
+          
+          font-size:16px;
+
```
Comment (oskargaw) [inline src/assets/scss/App.scss:261]: remove space here

File: src/assets/scss/App.scss [hunk 8]
```diff
@@ -118,3 +118,179 @@ th {
...
+          
+      }
+
+      .search{
+          padding: 15px 20px;
+          width: 200px;
+          border: 1px solid #3F3D56;
+          border-radius: 30px;
+          font-size: 16px;
+
```
Comment (oskargaw) [inline src/assets/scss/App.scss:270]: remove space here

File: src/assets/scss/App.scss [hunk 9]
```diff
@@ -118,3 +118,179 @@ th {
...
+      
+      }
+
+      .search:hover{
+          padding: 15px 20px;
+          width: 210px;
+          border: 1px solid #3F3D56;
+          border-radius: 30px;
+          font-size: 16px;
+
```
Comment (oskargaw) [inline src/assets/scss/App.scss:279]: remove space here

File: src/assets/scss/App.scss [hunk 10]
```diff
@@ -118,3 +118,179 @@ th {
...
+          padding: 15px 20px;
+          width: 210px;
+          border: 1px solid #3F3D56;
+          border-radius: 30px;
+          font-size: 16px;
+      
+      }
+
+      .littleBox{
+
```
Comment (oskargaw) [inline src/assets/scss/App.scss:283]: remove space here

File: src/assets/scss/App.scss [hunk 11]
```diff
@@ -118,3 +118,179 @@ th {
...
+          border: 1px solid #3F3D56;
+          border-radius: 30px;
+          font-size: 16px;
+      
+      }
+
+      .littleBox{
+         
+          display:flex;
+
```
Comment (oskargaw) [inline src/assets/scss/App.scss:285]: remove space here

File: src/assets/scss/App.scss [hunk 12]
```diff
@@ -118,3 +118,179 @@ th {
...
+          
+        }
+
+  }
+
+
+  @media (min-width: 768px) {
+      .h-md-100 { height: 100vh; }
+      
+
```
Comment (oskargaw) [inline src/assets/scss/App.scss:294]: remove space here

File: src/views/Category.vue [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+        </div>
+    </div>
+    
+    <!-- Second Half -->
+    
+    <div class="col-md-6 p-0 bg-white h-md-100 loginarea">
+        <div class="d-md-flex align-items-center h-md-100 p-5 justify-content-center">
+            
+
+
```
Comment (oskargaw) [inline src/views/Category.vue:27]: remove space here

File: src/views/Category.vue [hunk 14]
```diff
@@ -0,0 +1,55 @@
...
+                  <div class="col-md-5 myCol">Product</div>
+                  <div class="w-100"></div>
+                  <div class="col-md-5 myCol">Product</div>
+                  <div class="col-md-5 myCol">Product</div>
+                </div>
+              </div>
+          
+              
+  
+
```
Comment (oskargaw) [inline src/views/Category.vue:49]: remove space here

File: src/assets/scss/App.scss [hunk 15]
```diff
@@ -118,3 +118,179 @@ th {
...
+  border: 1px solid #3F3D56;
+  margin: 2px;
+}
+
+.text h3, h5{
+  color: #3F3D56;
+}
+
+
+  @media screen and (max-width: 1200px) {
```
Comment (oskargaw) [inline src/assets/scss/App.scss:210]: change '@media screen and'  to '@media only screen and'

File: src/assets/scss/App.scss [hunk 16]
```diff
@@ -118,3 +118,179 @@ th {
...
+          color:#fff;
+          font-family:Arial;
+          font-size:20px;
+          padding:5px 20px;
+          text-decoration:none;
+          
+      }
+    }
+
+  @media (max-width: 768px) {
```
Comment (oskargaw) [inline src/assets/scss/App.scss:240]: change '@media screen and' to '@media only screen and'

File: src/assets/scss/App.scss [hunk 17]
```diff
@@ -118,3 +118,179 @@ th {
...
+      .littleBox{
+         
+          display:flex;
+          
+        }
+
+  }
+
+
+  @media (min-width: 768px) {
```
Comment (oskargaw) [inline src/assets/scss/App.scss:291]: change '@media screen and' to '@media only screen and'

File: src/assets/scss/App.scss [hunk 18]
```diff
@@ -118,3 +118,179 @@ th {
   text-align: left;
   padding: 8px;
 }
+
+.bigBox{
```
Comment (szywyk) [inline src/assets/scss/App.scss:122]: Wrap whole category css into block
.category {
}

**Pull request closed without merging**